### PR TITLE
Connected SignUpView and UserRegPresenter

### PR DESCRIPTION
**Pull request by carolinexuxy**
Connected SignUpView and UserRegPresenter, made some minor changes in both files.

### CODE REVIEW

**Review by jay-ead — APPROVED** [review]
LGTM

**Review by whyismynamerudy — APPROVED** [review]
LGTM! I appreciate the changes you've made to LoginView and SignupView - I hope they work when we link it with the rest of the application. I'm also taking note of the Javadoc that you've included; however I do think it may help if you also describe what the function does in its Javadoc - you can look at the Javadoc within ChatRepoUseCase if you want to see an example. Other than that, looks good! Approved.

File: src/main/java/FrameworksDrivers/SignUpView.java [hunk 1]
```diff
@@ -71,21 +74,45 @@ public void sendPaths(Object[] paths){
...
+            this.layout.show(this.masterPanel, "userEditView");
+        }
     }
 
+
     @Override
     public void actionPerformed(ActionEvent evt) {
 
         if (evt.getSource() == regB.getButton()) {
+            SignUpView newView = new SignUpView(this.masterPanel, this.layout);
```
Comment (skoooch) [inline src/main/java/FrameworksDrivers/SignUpView.java:108]: GReat MVP structure!

**Review by skoooch — APPROVED** [review]
LGTM

**Pull request merged**